pyp5js/http/web_app.py

```diff
@@ -35,16 +35,17 @@ def add_new_sketch_view():
         sketch_name = slugify(request.form.get('sketch_name', '').strip(), separator='_')
         if not sketch_name:
             context['error'] = "You have to input a sketch name to proceed."
-        try:
-            files = commands.new_sketch(sketch_name)
-            template = 'new_sketch_success.html'
-            context.update({
-                'files': files,
-                'sketch_url': f'/sketch/{sketch_name}/',
-            })
-        except SketchDirAlreadyExistException:
-            path = SKETCHBOOK_DIR.joinpath(sketch_name)
-            context['error'] = f"The sketch {path} already exists."
+        else:
+            try:
```

Comment:
@berinhard had to add the `else` because after creating the test for the empty sketch_name scenario it failed because even though it falls in the `if` the try block is executed anyway and the error message is overwritten with `'The sketch {path} already exists.'` message

Reply:
For sure! Nice catch!